src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -134,6 +135,29 @@ void emberAfModeSelectClusterServerInitCallback(EndpointId endpointId)
             // Initialise currentMode to 0
             uint8_t currentMode = 0;
             status              = Attributes::CurrentMode::Get(endpointId, &currentMode);
+#ifdef EMBER_AF_PLUGIN_ON_OFF
+            // OnMode with Power Up
+            // If the On/Off feature is supported and the On/Off cluster attribute StartUpOnOff is present, with a
```

Comment:
The spec, btw, does not say anything about "StartUpOnOff" here.  It just says "and the On/Off cluster attribute is present".  Spec issue needed?

Reply:
The Spec has a reference pointing "On/Off cluster attribute" to the StartUpOnOff section. This is not possible with code comment.

Reply:
My point is the spec should explicitly say "StartUpOnOff" because there are versions of the spec where links don't work (e.g. hard-copy).

Reply:
pull request created: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/5064/files